src/contentful/ContentfulClientOptions.ts

```diff
@@ -1,10 +1,11 @@
 
 
 export interface ContentfulClientOptions {
-
     readonly accessToken: string;
     readonly spaceId: string;
+    readonly host?: string;
+    readonly headers?: object|string;
```

Comment:
Headers should probably be `Record<string, string> | string` instead of object, no?